server/base/src/main/java/org/apache/accumulo/server/security/handler/ZKPermHandler.java

```diff
@@ -401,6 +401,7 @@ public void initializeSecurity(TCredentials itw, String rootuser)
         Collections.singleton(NamespacePermission.ALTER_NAMESPACE));
     namespacePerms.put(Namespace.ACCUMULO.id(),
         Collections.singleton(NamespacePermission.ALTER_TABLE));
+    namespacePerms.put(Namespace.DEFAULT.id(), Collections.singleton(NamespacePermission.READ));
```

Comment:
The root user shouldn't have this permission by default. This namespace is for user tables, and there's no reason the root user should have read permission on the user tables by default.

Reply:
Right, so I'm assuming that the SYSTEM permission is required to read the instance/system configs, NamespacePermission.READ is required to read the namespace configs, and TablePermission.READ is required to read table property / configs.

Reply:
Addressed in 7c8df00. I removed this line from ZKPermHandler